advanced_survey/views.py

```diff
@@ -300,11 +301,54 @@ def get_surveyjs(request):
 
     return JsonResponse(surveyjs)
 
-def validate(question, answer):
+def validate(question, answer): # pylint: disable=R0911,R0912
```

Comment:
Can we implement this by raising exceptions instead of returning error messages? Alternatively, please change the name to `get_error` or something similar - it's a bit confusing to have `validate` return a truthy value for an error and a falsey value for valid date

Reply:
fixed.